service/completion.rkt

```diff
@@ -19,9 +18,11 @@
       (set! completions (list)))
 
     (define/override (walk-stx stx expanded-stx)
-      (define c (append (set->list (walk expanded-stx))
-                        (set->list (walk-module expanded-stx))))
+      (define c (walk expanded-stx))
```

Comment:
If we don't need to append, we also no need variable `c` then

Reply:
yes